Newfeature

jaredmael opened the pull request:
Added Registration Page.  Please get to Max so he can hook up to registration

jaredmael committed:
CSS static folder, homepage and login attempt

The CSS is in with the javascript, the homepage redirects to login, but
something is wrong as after pressing login button on login page, user
returned to homepage (whether login is correct or not leaving in endless
loop between the two pages.  something wrong in the Django views.

jaredmael committed:
CSS static folder added, Login updated, Homepage

CSS static folder was added.  Homepage added, Login page update with
CSS.  This all works as tested by another team member.

jaredmael committed:
Fixed Login Fail Message

Made the failed login message look nicer

jaredmael committed:
Added Registration Page and edited HomePage

Registration page added and Homepage registration button works.  Note
button hook up on submit from registration doesnt work yet because we
need to discuss how to handle this ASAP.  I can make a registration
accepted page but before I do that we need the hook up from registration
to admin acknowledgement page.

jaredmael committed:
Added button to create new projects in project view

On the projects page I added a create project button to be applied to
Project Manager.  For now it is visible to all users for testing
purposes.  As of right now it is not scripted to do anything.  It is
setup for those in charge of this step.

jaredmael committed:
Project Member form added.  Needs backend hookups

Working Project member UI.  To view go to base url/members

mraaa commented:
Hi Jared:

mraaa commented:
Hi Jared:

I always get 404 from background image by tracking error-log

Also, the registration page have some thing need to change:
1. The password field and confirm password field should not be a plain text.

Please try to correct it. Then I will test it again. Thank you very much !!

jaredmael committed:
Added thank you page for registering

Registration page now redirects to a thank you page if the form is
filled out correctly.

jaredmael committed:
Added a create Project Form popup with Cancel

Popup style Create project form.  Cancel button closes window.  Figuring
out the disable parent form (might be in iteration 2) but this is ready
for the create project scripting.  Form launched when create project
button clicked from the project list page.
